### PR TITLE
Install graphviz on xcode7.3 and xcode8.3 travis builds 

### LINKED ISSUE #255: "Go: master Xcode: xcode6.4" CI fails, apparently during graphviz install
Saw this in the travis check for #254.

```
Updating Homebrew...
==> Installing dependencies for graphviz: libtool, pkg-config, libpng, freetype, fontconfig, jpeg, libtiff, webp, gd
==> Installing graphviz dependency: libtool
==> Downloading https://homebrew.bintray.com/bottles/libtool-2.4.6_1.yosemite.bo
==> Pouring libtool-2.4.6_1.yosemite.bottle.tar.gz
==> Caveats
In order to prevent conflicts with Apple's own libtool we have prepended a "g"
so, you have instead: glibtool and glibtoolize.
==> Summary
🍺  /usr/local/Cellar/libtool/2.4.6_1: 70 files, 3.7MB
==> Installing graphviz dependency: pkg-config
==> Downloading https://homebrew.bintray.com/bottles/pkg-config-0.29.2.yosemite.
==> Pouring pkg-config-0.29.2.yosemite.bottle.tar.gz
🍺  /usr/local/Cellar/pkg-config/0.29.2: 11 files, 631KB
==> Installing graphviz dependency: libpng
==> Downloading https://downloads.sourceforge.net/libpng/libpng-1.6.34.tar.xz
==> Downloading from https://downloads.sourceforge.net/project/libpng/libpng16/1
==> ./configure --disable-silent-rules --prefix=/usr/local/Cellar/libpng/1.6.34
==> make
==> make test
Last 15 lines from /Users/travis/Library/Logs/Homebrew/libpng/03.make:
============================================================================
# TOTAL: 33
# PASS:  32
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to png-mng-implement@lists.sourceforge.net
============================================================================
make[2]: *** [test-suite.log] Error 1
make[1]: *** [check-TESTS] Error 2
make: *** [check-am] Error 2
Do not report this issue to Homebrew/brew or Homebrew/core!
These open issues may also help:
Formulae with opportunistic linkage issues https://github.com/Homebrew/homebrew-core/issues/16531
php: 7.1 (new formula) https://github.com/Homebrew/homebrew-core/pull/16067
Error: You are using macOS 10.10.
We (and Apple) do not provide support for this old version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.
Error: You are using macOS 10.10.
We (and Apple) do not provide support for this old version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.
The command "if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install graphviz; fi" failed and exited with 1 during .
Your build has been stopped.
```

### PR DESCRIPTION
**Pull request by nolanmar511**
PR #270 want meant to stop installing graphviz on xcode6.4 to fix #255.

However, #270 also stopped graphviz from being installed on  xcode7.3 and xcode8.3 (issue #317).

This PR modifies .travis.yml so graphviz is installed on   xcode7.3 and xcode8.3.

Completed build: https://travis-ci.org/nolanmar511/pprof/builds/339568567
I confirmed that TestWebInterface ran on all linux builds, all xcode7.3 and xcode8.3 builds, but not on any xcode6.4 builds.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -52,8 +50,8 @@ addons:
       
 before_install:
   - go get -u github.com/golang/lint/golint honnef.co/go/tools/cmd/...   
-  - if (("$TRAVIS_OS_NAME" == "osx") && (!"$SKIP_GRAPHVIZ")); then brew update          ; fi
-  - if (("$TRAVIS_OS_NAME" == "osx") && (!"$SKIP_GRAPHVIZ")); then brew install graphviz; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" && (-z $SKIP_GRAPHVIZ) ]]; then brew update          ; fi
```
Comment (aalexand) [inline .travis.yml]: I think ( and ) are not needed.

**Comment by nolanmar511** [comment]
Just updated description with completed travis build

**Comment by aalexand** [comment]
@nolanmar511 It would be good to include more detail on what you checked when any manual testing is done. "Confirmed it works" tells little :)

File: .travis.yml (hunk 1 above)
Comment (nolanmar511) [reply .travis.yml]: This seems to work.
https://travis-ci.org/nolanmar511/pprof/builds/339596412
Build 301.8 has output after `if [[ "$TRAVIS_OS_NAME" == "osx" && -z $SKIP_GRAPHVIZ ]]; then brew install graphviz; fi` that's

Comment (nolanmar511) [reply .travis.yml]: Done.

**Pull request merged**